models/lung_nodule_ct_detection/configs/inference_raw.json

```diff
@@ -0,0 +1,55 @@
+{
+    "data_list_file_path": "$@bundle_root + '/LUNA16_datasplit/original/dataset_fold0.json'",
+    "data_file_base_dir": "/home/canz/Projects/datasets/LUNA16/93176/Images",
+    "preprocessing": {
+        "_target_": "Compose",
+        "transforms": [
+            {
+                "_target_": "DeleteItemsd",
+                "keys": [
+                    "box",
+                    "label"
+                ]
+            },
+            {
+                "_target_": "LoadImaged",
+                "keys": "image",
+                "meta_key_postfix": "meta_dict",
+                "reader": "itkreader",
```

Comment:
I feel the only difference in this new config is this line, no need to override all the transforms.
Maybe you can just add a flag in the original `inference.json`? For example:
```json
"resampled_image": "true",
...
"reader": "$'nibabelreader' if @resampled_image else 'itkreader'",
...
```

What do you think?
CC @yiheng-wang-nv @wyli .

Thanks.

Reply:
Agree. Similar for the `Spacingd` transform, can add a flag and make use of `_disabled_` to control whether to use this transform. Can refer to [here](https://github.com/Project-MONAI/model-zoo/pull/163/files#diff-53004b9136c53f2fc7834c750c784abc55fc3161cfe1c53c1143dde5ae581a23R127)

Reply:
There are at least three places to switch. I was afraid it is not that convenient to use `"_disabled_"`

Reply:
Hi @yiheng-wang-nv , I don't see any difference in the `Spacingd` transform, what do you mean to change for `resampled or raw`?

Thanks.

Reply:
There is a chnge. When apply to resampled image, we should not include spacing